scripts/configure-installer-tests-cluster.sh

```diff
@@ -74,6 +78,9 @@ done
 # Workarounds - Note we should find better soulutions asap
 ## Missing wildfly in OpenShift Adding it manually to cluster Please remove once wildfly is again visible
 oc apply -n openshift -f https://raw.githubusercontent.com/openshift/library/master/arch/x86_64/community/wildfly/imagestreams/wildfly-centos7.json
+## Applying pull sceret to the namespace which will be used for pulling images from authenticated registry
+oc get secret pull-secret -n openshift-config -o yaml | sed 's/openshift-config/openjdk-11-rhel8/g' | oc apply -f -
+oc get secret pull-secret -n openshift-config -o yaml | sed 's/openshift-config/nodejs-12-rhel7/g' | oc apply -f -
```

Comment:
I would suggest lets run it in a loop considering the assumption if the private registry image support will increase.

Reply:
Looping is a good idea when we have to iterate over more than one items.

Reply:
Done